mtwitter.js

```diff
@@ -0,0 +1,291 @@
+var Twitter = require('twitter');
+var Request = require('request');
+var fs = require('fs');
+var Buffer = require('buffer').Buffer;
+
+const TWITTER_PROFILE_INTERVAL = 60000;//ms;
+
+var MTwitter = function (bridge, config) {
+  this.app_auth = config.app_auth;
```

Comment:
It seems that you only use `config` to pull out `app_auth`. Perhaps you should just pass in an `app_auth`? If you're planning to add more things to the config in the future, then keep it as `config`, but definitely add JSDoc on this constructor to state what properties you expect the `config` to have.